### DISCUSSION TITLE
Is it possible to add a scrollbar to the navigation table?

### DISCUSSION THREAD

**Discussion by daryafret** [mkdocs/mkdocs#2499] [General]
Not all headers fit entirely on the navigation table.. Is there any parameteres to fix them?

![image](https://user-images.githubusercontent.com/47778180/125401750-3ce61300-e3bc-11eb-8b68-3740603abbcb.png)

**Comment by ultrabug** [comment]
@squidfunk maybe you'd know some frontend trick we could use to improve that?

**Reply by squidfunk** [reply]
This is currently not handled. It might be solvable by allowing the tabs container to scroll with some custom CSS. It's however nothing I plan to support in Material for MkDocs, because it would cutoff content which might be undiscoverable for the user. Horizontal scrolling on large screens is always a kind of pain.

The general advise here is to restructure the navigation hierarchy so the tabs don't overflow.

**Reply by daryafret** [reply]
ok. thanks for answer

**Comment by ultrabug** [comment]
@daryafret could you please provide the exact theme + version you're using?
